data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -64,6 +68,9 @@
     <ask-list config:type="list"/>
     <mode>
       <confirm config:type="boolean">false</confirm>
+      % if ($check_var->('VERSION', '15-SP4')) {
+      <final_reboot config:type="boolean">true</final_reboot>
```

Comment:
It will fix this http://openqa.qam.suse.cz/tests/44419#step/installation/7. 
15-SP4 login screen was messed up after installation. The fix will reboot one more time to get a clean login screen.

Reply:
http://openqa.qam.suse.cz/tests/44291#step/installation/8

Reply:
It's ok that it can get around the console problem anyway.